command/server/config_test_helpers.go

```diff
@@ -456,6 +457,41 @@ func testLoadConfigFile(t *testing.T) {
 	}
 }
 
+func testUnknownFieldValidation(t *testing.T) {
+	config, err := LoadConfigFile("./test-fixtures/config.hcl")
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+
+	expected := []configutil.ConfigError{
+		{
+			Problem:  "unknown field bad_value found in configuration",
+			Position: token.Pos{},
+		},
+	}
+	errors := config.Validate("./test-fixtures/config.hcl")
+
+outer:
+	for _, er1 := range errors {
+		for _, ex := range expected {
+			// Only test the string, pos may change
+			if ex.Problem == er1.Problem {
+				continue outer
```

Comment:
could these be refactored to just `break` and not have the tags?

Also, what are those tags called? I've only seen them with `GOTO`

Reply:
And I ask about refactoring (at least for this loop) because i think it might be easier for another eng to understand down the road (i think it would have taken me less time to get it)

Reply:
They're block labels.  The idea there was if you just break on the inner one you'll hit the t.Fatal, which isn't what we want.  That said I can do it with a found bool instead.

Reply:
cool, i haven't seen that used before! closest i've seen is with goto. i'm gonna read about them!

do whatever you think is most readable, i'm not too concerned about this

Reply:
https://medium.com/golangspec/labels-in-go-4ffd81932339